BenniTask.py

```diff
@@ -0,0 +1,58 @@
+# Defining the minimum and maximum withdrawal amounts
+min_withdrawal = 5 
+max_withdrawal = 1000 
+
+# Initializing the Customer balance
+customer_balance = 1000  # Initial balance 
+
+# Optional input your balance
+customer_balance_input = float(input("Enter your Balance: ",)) 
```

Comment:
becareful here, the user might input something that cant be converted to a float, eg a user can input "john"